models/prices/arbitrum/prices_arbitrum_tokens.sql

```diff
@@ -93,6 +93,7 @@ FROM
     ('ush-unsheth', 'arbitrum', 'unshETH', 0x0ae38f7e10a43b5b2fb064b42a2f4514cba909ef, 18),
     ('wad-warden','arbitrum','WAD',0x6374d87c5a48c41b309a1ab7b12eeb4fe30d8d8a,18),
     ('wbtc-wrapped-bitcoin', 'arbitrum', 'WBTC', 0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f, 8),
+    ('weeth-wrapped-eeth','arbitrum','weETH',0x35751007a407ca6feffe80b3cb397736d2cf4dbe,18),
```

Comment:
it looks like this address already exists?
https://github.com/duneanalytics/spellbook/blob/main/models/prices/arbitrum/prices_arbitrum_tokens.sql#L211